internal/common/filetrustsource.go

```diff
@@ -41,11 +50,13 @@ func NewTrustFileSource(domainPaths map[string][]string) (TrustFileSource, error
 	return source, nil
 }
 
-func (source *TrustFileSource) TrustedCertificates() map[string][]*x509.Certificate {
+// TrustedCertificates returns our current maps of domains to certificates. This is a
```

Comment:
I take it this is just a stub before making it thread safe?

Reply:
Yes, that is correct.